torch/sparse/_triton_ops.py

```diff
@@ -463,19 +460,42 @@ def check(cond, msg):
                 f"but got bsr.dim() == {bsr.dim()} and dense.dim() == {dense.dim()}.",
             )
 
+            m, kl = bsr.shape[-2:]
+            kr, n = dense.shape[-2:]
+
             check(
                 kl == kr,
                 "bsr_dense_mm(): argument sizes are not compatible for matrix multiplication, "
                 f"got bsr.shape[-1] == {kl} which is not equal to dense.shape[-2] == {kr}.",
             )
 
-            row_block = bsr.values().shape[-2]
+            row_block, col_block = bsr.values().shape[-2:]
             check(
                 not n % row_block,
```

Comment:
Nit (same as below, not sure about `not` definition for Python and operator precedence, wheres `x % y == z` is much more straightforward, isn't it?
```suggestion
               n % row_block == 0,
```